### ISSUE TITLE
fastify specify a route for register 

### ISSUE THREAD

**Issue by dsun9** [fastify/fastify#532]
Hi, I'd like to know if we can specify a route for register. 

I have a route that I don't want the following, which slows it down. but other routes need them
fastify.register(fastifyCookie)
fastify.register(require('fastify-static')
fastify.register(require('fastify-multipart'))
fastify.register(require('fastify-formbody')

**Comment by mcollina** [comment]
I don’t understand what you want to achieve, can you rephrase?

**Comment by jsumners** [comment]
```js
const fastify = require('fastify')()

fastify.register(function (instance, options, next) {
  fastify.get('/foo', function (req, reply) {
    reply.send('no plugins affecting this route')
  })
  next()
})

fastify.register(function (instance, options, next) {
  instance
    .regiser(fastifyCookie)
    .register(require('fastify-static'))
    .register(require('fastify-multipart'))
    .register(require('fastify-formbody'))
    .get('/bar', function (req, reply) {
      reply.send('several plugins affect this route')
    })
  next()
})
```

**Comment by dsun9** [comment]
Thanks jsumners, let me try that.
Thanks mcollina, jsumners' post is what I want to achieve

**Comment by dsun9** [comment]
I got this error
app.js:12
    .regiser(require('fastify-cookie'))
     ^

TypeError: instance.regiser is not a function

**Comment by jsumners** [comment]
@dsun9 it is very clearly a typo. Don't copy and paste an example from someone who has no idea what your actual code looks like.

**Comment by dsun9** [comment]
Thanks very much
